Leanplum-iOS-SDK-Example/Configure.h

```diff
@@ -16,7 +16,7 @@
 #define LP_PRODUCTION_KEY @"prod_A1c7DfHO6XTo2BRwzhkkXKFJ6oaPtoMnRA9xpPSlx74"
 #define LP_DEVELOPMENT_KEY @"dev_Bx8i3Bbz1OJBTBAu63NIifr3UwWqUBU5OhHtywo58RY"
 
-#define LP_API_HOST_NAME @"127.0.0.1:3000"
+#define LP_API_HOST_NAME @"10.0.2.2:3000"
```

Comment:
why?

Reply:
To be the same as Android Example app, and It works with api server when set to 10.0.2.2:3000.

Reply:
why does it need to be same?  why we can't use 127.0.0.1 for iOS?

Reply:
Doesnt connect to API server, connects with 10.0.2.2:3000, hence why ben changed it for android i believe. Im going to merge my changes.

Reply:
how tests worked before?

Reply:
we do not need this changes. For iOS localhost should work, for Android it special situation.

Reply:
@Enuviel feel free to test it yourself, I have and I discovered that 127.0.0.1:3000 does not work and that 10.0.2.2:3000 does work.

Reply:
I reverted it so that we can move on